### ISSUE TITLE
Accessibility: the tabs on the bottom are missing labels causing voiceover to read tab as the tab name 

### ISSUE THREAD

**Issue by markcartertm** [dangwu/Octal#187]
On iOS go to iOS settings accessibility, then voiceover on. Now go to the app and put your fingers on the tabs at the bottom. Note that voiceover will read the word tab for each of the tabs rather than what is the name or the purpose of the tab

**Comment by dangwu** [comment]
Fixing soon!

**Comment by dangwu** [comment]
This should be fixed with 3.34.0, released moments ago!
